csharp-basics/exercises/Loops/Loops/Exercise2/Program.cs

```diff
@@ -13,8 +13,12 @@ static void Main(string[] args)
             /*
             todo - complete loop to multiply i with itself n times, it is NOT allowed to use Math.Pow()
             */
-            for (i = 0; i < n; i++)
+            int res;
+
+            for (i = 0; i <= n; i++)
             {
+                res = i * i;
```

Comment:
vai tad i pirmajā reizē nav 0?

Reply:
Uzdevumā jau bija uzsakts LOOPs no 0. Es to atstāju.